1.3/Patches/z_add_chitchat_thoughts.xml

```diff
@@ -55,6 +67,7 @@
 												<li>thought_atewotable_construction_unskilled(RECIPIENT_construction_level&lt;4)-></li>
 											<!-- ===Recipient is skilled in construction=== -->
 												<li>thought_atewotable_construction_unskilled(RECIPIENT_construction_level>=4)->I hope you won't let this happen again, [recipient].</li>								
+												<li>thought_atewotable_construction_unskilled(RECIPIENT_construction_level>=4)->You should build one for the good of the group, [RECIPIENT_nameDef].</li>
```

Comment:
Shouldn't this be [recipient]? Or is this a "negative opinion" snippet?

Reply:
Fixing!